sys-libs/hwloc/hwloc-1.11.8.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="The Hardware Locality library"
+DESCRIPTION="The Hardware Locality (hwloc) software project aims at easing the \
+process of discovering hardware resources in parallel architectures. It offers \
+command-line tools and a C API for consulting these resources, their locality, \
+attributes, and interconnection. hwloc primarily aims at helping \
+high-performance computing (HPC) applications, but is also applicable to any \
+project seeking to exploit code and/or data locality on modern computing \
+platforms."
+HOMEPAGE="http://www.open-mpi.org/projects/hwloc/"
+COPYRIGHT="1998-2011 Free Software Foundation, Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://www.open-mpi.org/software/hwloc/v1.11/downloads/hwloc-${portVersion}.tar.bz2"
+CHECKSUM_SHA256="ecef10265084d9123dc0b03d82ff5ee06fbf5ba755a912e7511ae590c9ee594d"
+PATCHES="hwloc-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	hwloc$secondaryArchSuffix = $portVersion
+	lib:libhwloc$secondaryArchSuffix = 5.7.5 compat >= 5
+	cmd:lstopo$secondaryArchSuffix = $portVersion
+	cmd:lstopo_no_graphics$secondaryArchSuffix = $portVersion
+	cmd:hwloc_annotate$secondaryArchSuffix = $portVersion
+	cmd:hwloc_assembler$secondaryArchSuffix = $portVersion
+	cmd:hwloc_assembler_remote$secondaryArchSuffix = $portVersion
+	cmd:hwloc_bind$secondaryArchSuffix = $portVersion
+	cmd:hwloc_calc$secondaryArchSuffix = $portVersion
+	cmd:hwloc_compress_dir$secondaryArchSuffix = $portVersion
+	cmd:hwloc_diff$secondaryArchSuffix = $portVersion
+	cmd:hwloc_distances$secondaryArchSuffix = $portVersion
+	cmd:hwloc_distrib$secondaryArchSuffix = $portVersion
+	cmd:hwloc_dump_hwdata$secondaryArchSuffix = $portVersion
+	cmd:hwloc_gather_topology$secondaryArchSuffix = $portVersion
+	cmd:hwloc_info$secondaryArchSuffix = $portVersion
+	cmd:hwloc_ls$secondaryArchSuffix = $portVersion
+	cmd:hwloc_patch$secondaryArchSuffix = $portVersion
+	cmd:hwloc_ps$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	hwloc${secondaryArchSuffix}_devel = $portVersion
+	devel:libhwloc$secondaryArchSuffix = $portVersion
```

Comment:
Please replace
```
	devel:libhwloc$secondaryArchSuffix = $portVersion
```
by
```
	devel:libhwloc$secondaryArchSuffix = $libVersion
```
or maybe by
```
	devel:libhwloc$secondaryArchSuffix = $libVersion compat >= 5
```
I don't know which is most appropriate. Help welcome ;-)